inst/rmarkdown/templates/ims/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,422 @@
+---
+journal: aoas # aap, aop, aos, sts
+title: A sample article with some additional note
+thanks: # delete thanks field if not require, here notes for title
+  - ref: T1 
+    text: A sample additional note to the title.
+runtitle: A sample running head title
+author:
+  - name: First 
+    surname: Author
+    email: \email{first@somewhere.com}
+    label: e1
+    addressLabel: A
+    sepNext: "," # use "," or \and; the last author not need this field
+  - name: Second
+    surname: Author
+    email: \email{second@somewhere.com}
+    label: e2,mark
+    addressLabel: B
+    sepNext: \and
+  - name: Third
+    surname: Author
+    email: \email{third@somewhere.com}
+    label: e3,mark
+    addressLabel: B
+affiliation:
+  - label: A
+    name: Department, University or Company Name
+    authorsLabels: e1
+  - label: B
+    name: Department, University or Company Name
+    authorsLabels: e2,e3
+abstract: |
+  The abstract should summarize the contents of the paper.
+  It should be clear, descriptive, self-explanatory and not longer
+  than 200 words. It should also be suitable for publication in
+  abstracting services. Formulas should be used as sparingly as
+  possible within the abstract. The abstract should not make
+  reference to results, bibliography or formulas in the body
+  of the paper---it should be self-contained.
+
+  This is a sample input file.  Comparing it with the output it
+  generates can show you how to produce a simple document of
+  your own.
+keyword-subClass: |
+  <!-- \begin{keyword}[class=MSC2020] % Uncomment for aap, aop, aos journals -->
+  <!-- \kwd[Primary ]{00X00} -->
+  <!-- \kwd{00X00} -->
+  <!-- \kwd[; secondary ]{00X00} -->
+  <!-- \end{keyword} -->
```

Comment:
If I understand correctly, this part is only for 3 of the five journal ? 

I think it is fine as you set it but maybe the key should also be commented not the content. Like 
```suggestion
# Uncomment below for aap, aop, aos journals
# keyword-subClass: |
#  \begin{keyword}[class=MSC2020]
#  \kwd[Primary ]{00X00}
#  \kwd{00X00}
#  \kwd[; secondary ]{00X00}
#  \end{keyword}
```

Also, maybe keep everything lowercase for consistency ? `keyword-subclass` ? 

The way to do it without commenting / commenting would be to have the journal being a variable in the function 
````r
output:
  ims_article:
    journal: aap
````
You could then check the journal name and set
* the journal variable for the template as it is now
* Set a `keyword-subclass` variable that would activate this part in the template. 

It is not necessary to do this change, but just sharing how I would do it.

Reply:
Thanks for the tip. Now the `ims_article()` includes the new parameter and sets a new variable that activates this part in the template. Also, the template variable changed to `keyword-subclass`